licenses.d2iq.yaml

```diff
@@ -385,32 +385,27 @@ resources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/stakater/Reloader
-  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/cloud-native/gpu-operator-validator:v24.3.0-d2iq.0
+  - container_image: nvcr.io/nvidia/cloud-native/gpu-operator-validator:v24.6.2
     sources:
       - license_path: validator/LICENSE
-        ref: ${image_tag%-d2iq.0}
+        ref: ${image_tag}
         url: https://github.com/NVIDIA/gpu-operator
-  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/gpu-operator:v24.3.0-d2iq.0
+  - container_image: nvcr.io/nvidia/gpu-operator:v24.6.2
     sources:
       - license_path: LICENSE
-        ref: ${image_tag%-d2iq.0}
+        ref: ${image_tag}
         url: https://github.com/NVIDIA/gpu-operator
-  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/k8s-device-plugin:v0.15.0-ubi8-d2iq.0
+  - container_image: nvcr.io/nvidia/k8s-device-plugin:v0.16.2
     sources:
       - license_path: LICENSE
-        ref: ${image_tag%-ubi8-d2iq.0}
+        ref: ${image_tag%}
         url: https://github.com/NVIDIA/k8s-device-plugin
-  - container_image: nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubuntu20.04
+  - container_image: nvcr.io/nvidia/k8s/container-toolkit:v1.16.2-ubuntu20.04
     sources:
       - license_path: LICENSE
         ref: ${image_tag%-ubuntu20.04}
         url: https://github.com/NVIDIA/nvidia-container-toolkit
-  - container_image: ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubi8-d2iq.0
```

Comment:
Lets keep the `1.16.0-ubi8` version in the bundle via `extra-images.txt` by adding the

```
nvcr.io/nvidia/k8s/container-toolkit:{{ regexReplaceAllLiteral "-.+$" .Values.toolkit.version "" }}-ubi8
```

Reply:
@shubham2g can you add back the mapping for the ub8 image?